src/main/java/io/jenkins/plugins/opentelemetry/JenkinsOpenTelemetryPluginConfiguration.java

```diff
@@ -163,6 +163,17 @@ public JenkinsOpenTelemetryPluginConfiguration() {
         load();
     }
 
+    @Override
```

Comment:
I would add a comment pointing to the issue and why this is needed

```suggestion
    // needed by CloudBees HA see https://github.com/jenkinsci/opentelemetry-plugin/issues/1156
    @Override
```